Pokedex WatchKit Extension/PokemonDetail.swift

```diff
@@ -21,13 +21,13 @@ struct PokemonDetail: View {
                 }
             }
             Section(header: Text("Types")) {
-                ForEach(pokemon.pkType) { pkType in
-                    NavigationLink(destination: TypeDetail(pkType: pkType)) {
+                ForEach(pokemon.types, id: \.self) { type in
```

Comment:
as you can see, you don't need to create structs, for them to be identifiable. You can specify here that the string itself is already identifiable. Or, in case of the pokemon, you can put like this `List(pokemonData, id: \.name)`

Reply:
Their names are not unique, though

Reply:
but point taken on the type